src/components/VmUserMessages/index.js

```diff
@@ -84,14 +84,14 @@ class VmUserMessages extends React.Component {
           <NotificationDrawer.Title onCloseClick={this.handleToggle} onExpandClick={this.handleExpand} />
           <NotificationDrawer.PanelBody className={style['panel-body']}>
             {messagesList}
-            <NotificationDrawer.PanelAction>
+            { messagesCount > 0 && <NotificationDrawer.PanelAction>
```

Comment:
The logic looks good.

In terms of style, I think it would be more consistent with the rest of the code to have the tag indented on the following line like this:
````js
            { messagesCount > 0 && 
              <NotificationDrawer.PanelAction>
                <NotificationDrawer.PanelActionLink data-toggle='clear-all'>
                  <Button bsStyle='link' onClick={onClearMessages}>
                    <Icon type='pf' name='close' />
                    { msg.clearAll() }
                  </Button>
                </NotificationDrawer.PanelActionLink>
              </NotificationDrawer.PanelAction>
            }
````